docs/pages/guides/building-for-tv.mdx

```diff
@@ -27,25 +27,29 @@ The necessary changes to the native Android and iOS files are minimal and can be
   - The Xcode project is modified to target tvOS instead of iOS
   - The splash screen (**SplashScreen.storyboard**) is modified to work on tvOS
 
-## Quick start
-
-The fastest way to generate a new project is described in the [TV Example](https://github.com/expo/examples/tree/master/with-tv) within the Expo examples repository.
-
-## Integration with an existing Expo project
-
-If you have an existing Expo project, the following walkthrough describes the steps required to modify an Expo project for TV.
+## System requirements for TV development
 
-You will need:
-
-- An Expo project using SDK 50 or later.
 - **Android TV**:
+  - [Node.js (LTS)](https://nodejs.org/en/) on macOS or Linux.
```

Comment:
Should we add "Windows" here?

Reply:
It might work, but I have not personally tested that.